samples/listCdnKeys.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2022, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(projectId, location) {
+  // [START video_stitcher_list_cdn_keys]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // projectId = 'my-project-id';
+  // location = 'us-central1';
+
+  // Imports the Video Stitcher library
+  const {VideoStitcherServiceClient} =
+    require('@google-cloud/video-stitcher').v1;
+  // Instantiates a client
+  const stitcherClient = new VideoStitcherServiceClient();
+
+  async function listCdnKeys() {
+    const [cdnKeys] = await stitcherClient.listCdnKeys({
```

Comment:
This method supports automatic pagination, and will keep requesting keys until the whole list is retrieved, and might hit the request quota. We recommend using async iteration (`listCdnKeysAsync`) as described [here](https://github.com/googleapis/gax-nodejs/blob/main/client-libraries.md#auto-pagination). Same applies to other paginated methods (`listVodStitchDetails`, `listVodAdTagDetails`, `listLiveAdTagDetails`, `listSlates`).

Reply:
Done